### PR TITLE
[MME] Add an empty spgw ue conversion test

### PR DESCRIPTION
**Pull request by themarwhal**
## Summary
Adding a basic basecase UE conversion test for SPGW. 
(Will write a more involved testcase in future PRs)
Some caveats
* This test only works with Alex's changes
	* https://github.com/magma/magma/pull/6417
	* https://github.com/magma/magma/pull/6449/

## Test Plan
Once the PRs above have merged, I'll run basic unit tests and s1ap tests

## Additional Information

- [ ] This change is backwards-breaking

Signed-off-by: Alex Rodriguez <ardzoht@gmail.com>

### CODE REVIEW

**Comment by ardzoht** [comment]
Hopefully #6449 helps here

**Review by ulaskozat — APPROVED** [review]
LGTM

**Comment by ekowtaylor** [comment]
@themarwhal is this still a draft? Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Comment by themarwhal** [comment]
> @themarwhal is this still a draft? Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

Yes it's still pending

**Pull request closed without merging**
